controllers/articles-controllers.js

```diff
@@ -1,10 +1,24 @@
-const { selectArticleById } = require("../models/articles-models");
+const {
+  selectArticles,
+  selectArticleById,
+} = require("../models/articles-models");
+
+exports.getArticles = (request, response, next) => {
+  const sort = "created_at ";
+  const order = "desc";
+
+  selectArticles(sort, order)
+    .then((articles) => {
+      response.status(200).send({ articles });
+    })
+    .catch((err) => next(err));
+};
 
 exports.getArticleById = (request, response, next) => {
-    const { article_id } = request.params;
-    selectArticleById(article_id)
-      .then((article) => {
-        response.status(200).send({ article });
-      })
-      .catch((err) => next(err));
-  };
\ No newline at end of file
+  const { article_id } = request.params;
```

Comment:
There seems to be a bit of a mix of tasks going on here. This is supposed to be task 5, all articles, but there are parts of the previous task here

Reply:
Hi August, it's because I waited for task 4 feedback, completed merging and then started task 5, as I wanted to take a rest while waiting.